dashboard/src/features/projects/overview/components/OverviewUsage/OverviewUsage.tsx

```diff
@@ -105,14 +105,24 @@ export function OverviewUsageMetrics() {
   // metrics for functions
   const usedFunctions = functionsInfoData?.app.metadataFunctions.length || 0;
   const totalFunctions = currentProject?.plan?.isFree ? 10 : 50;
+  const functionsDuration = projectMetrics?.functionInvocations.value || 0;
```

Comment:
I think this should be:

```suggestion
  const functionsDuration = projectMetrics?.functionsDuration.value || 0;
```

also, keep in mind the value returned is in seconds.